packages/main/src/services/config/index.ts

```diff
@@ -39,7 +39,8 @@ class Config {
     this.youtubeSearch = 'https://www.googleapis.com/youtube/v3/search?part=id,snippet&type=video&maxResults=50&q=';
     this.supportedFormats = _.uniq(pkg.build.fileAssociations.map(({ ext }) => ext));
 
-    const iconPath = path.resolve(__dirname, this.isProd() ? 'resources' : '../resources/media');
+    // const iconPath = path.resolve(__dirname, this.isProd() ? 'resources' : '../resources/media');
+    const iconPath = path.resolve(__dirname, this.isProd() ? '../../../resources/media' : '../resources/media');
```

Comment:
Thanks for getting to the bottom of this. Could you please delete the commented out line?

Reply:
I removed that comment.

Thanks for reviewing this.